Assets/UniMic/Runtime/Editor/MicEditor.cs

```diff
@@ -0,0 +1,58 @@
+#if UNITY_EDITOR
+
+using UnityEditor;
+using UnityEngine;
+
+namespace Adrenak.UniMic {
+	[CustomEditor(typeof(Mic))]
+	public class MicEditor : Editor {
+		private bool showInfo;
+
+		public override void OnInspectorGUI() {
+			base.OnInspectorGUI();
+
+			var mic = (Mic)target;
+
+			// Present a dropdown menu listing the possible styles
+			EditorGUILayout.BeginHorizontal();
+			EditorGUILayout.PrefixLabel("Select Device");
+			if (EditorGUILayout.DropdownButton(new GUIContent(mic.CurrentDeviceName), FocusType.Keyboard)) {
+				var menu = new GenericMenu();
+
+				foreach (var device in mic.Devices)
+					menu.AddItem(new GUIContent(device), mic.Devices[mic.CurrentDeviceIndex] == device,
+						OnDeviceSelected, device);
+
+				menu.ShowAsContext();
+			}
+
+			EditorGUILayout.EndHorizontal();
+
+			// Display a button to reset to the system's default mic
+			if (EditorGUILayout.LinkButton("Switch to System's Default Device"))
```

Comment:
Change EditorGUILayout.LinkButton to GUILayout.Button as it is not available on pre Unity 2021